website/static/css/custom.css

```diff
@@ -0,0 +1,16 @@
+/* your custom css */
+
+@media only screen and (min-device-width: 360px) and (max-device-width: 736px) {
+}
+
+@media only screen and (min-width: 1024px) {
+}
+
+@media only screen and (max-width: 1023px) {
+}
+
+@media only screen and (min-width: 1400px) {
+}
+
+@media only screen and (min-width: 1500px) {
```

Comment:
why are all of these in here?

Reply:
https://github.com/Microsoft/fast-dna/pull/1175#discussion_r237646930